[doc] schema metrics doc (#14901)
apache/pulsar: site2/docs/reference-metrics.md

| jetty_stats_seconds | Gauge | Time in seconds stats have been collected for. |
| jetty_responses_bytes_total | Counter | Total number of bytes across all responses. |

### Schema metrics

> For a functions-worker running separately from brokers, its schema metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.

All the schema metrics are labelled with the following labels:

- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.

| Name | Type | Description |
|---|---|---|

> | pulsar_schema_del_ops_failed_count | Counter | Number of delete schema failed operations. |

Reviewer: ```suggestion
| pulsar_schema_del_ops_failed_count | Counter | Number of operations that delete schemas |
```
do you intend to mean this？
if so, could you please help update other occurencies? thanks

> | pulsar_schema_del_ops_failed_count | Counter | Number of operations that delete schemas failed. |

Reviewer 2: ```suggestion
| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
```

| pulsar_schema_get_ops_failed_count | Counter | Number of failed operations to get schemas. |

> | pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to put schemas. |

Reviewer 2: ```suggestion
| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to send schemas. |
```

> | pulsar_schema_compatible_count | Counter | Number of schema compatible. |

Reviewer 2: ```suggestion
| pulsar_schema_compatible_count | Counter | Number of compatible schemas. |
```

> | pulsar_schema_incompatible_count | Counter | Number of schema incompatible. |

Reviewer 2: ```suggestion
| pulsar_schema_incompatible_count | Counter | Number of incompatible schemas. |
```

> | pulsar_schema_del_ops_latency | Summary | Latency of operations which delete schema succeed. |

Reviewer 2: ```suggestion
| pulsar_schema_del_ops_latency | Summary | Latency of successful operations to delete schemas. |
```

This sentence pattern applies to others.

> | pulsar_schema_get_ops_latency | Summary | Latency of successful operations to get schema. |

Reviewer 2: ```suggestion
| pulsar_schema_get_ops_latency | Summary | Latency of successful operations to get schemas. |
```

> | pulsar_schema_put_ops_latency | Summary | Latency of successful operations to put schema. |

Reviewer 2: ```suggestion
| pulsar_schema_put_ops_latency | Summary | Latency of successful operations to send schemas. |
```
